docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.8'
+
+services:
+  attack-flow:
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+    ports:
+      - 8081:80
+    volumes:
+      - /etc/attackflowbuilder/data:/usr/share/nginx/html/corpus
```

Comment:
I don't want to store the corpus inside of `/etc`. This should be a named volume, and if somebody wants to mount a local path, they can customize their docker compose file.

Reply:
@banzo Can you please change this to a named volume?

Reply:
I removed the volume, it does not make sense when curling the docker-compose.yml